draft-ietf-ppm-dap.md

```diff
@@ -191,81 +191,79 @@ seen in the clear by any server.
 
 {::boilerplate bcp14-tagged}
 
-The following terms are used:
-
 Aggregate result:
-: The output of the aggregation function over a given set of reports.
+: The output of the aggregation function over a given set of measurements and
+  aggregation parameter. As defined in {{!VDAF=I-D.draft-irtf-cfrg-vdaf-03}}.
 
 Aggregate share:
-: A share of the aggregate result emitted by an aggregator. Aggregate shares are
-  reassembled by the collector into the final output.
+: A share of the aggregate result emitted by an Aggregator. Aggregate shares are
+  reassembled by the Collector into the aggregate result, which is the final
+  output of the aggregation function. As defined in {{!VDAF}}.
 
 Aggregation function:
-: The function computed over the users' inputs.
+: The function computed over the Clients' measurements. As defined in {{!VDAF}}.
+
+Aggregation parameter:
+: Parameter used to prepare a set of measurements for aggregation (e.g., the
+  candidate prefixes for Poplar1 from {{Section 8 of !VDAF}}). As defined in
+  {{!VDAF}}.
 
 Aggregator:
-: An endpoint that runs the input-validation protocol and accumulates input
-  shares.
+: An endpoint that recieves input shares from Clients and validates and
+  aggregates them with the help of the other Aggregators. As defined in
+  {{!VDAF}}.
 
 Batch:
-: A set of reports that are aggregated into an output.
+: A set of reports that are aggregated into an aggregate result.
 
 Batch duration:
 : The time difference between the oldest and newest report in a batch.
 
 Batch interval:
-: A parameter of the collect or aggregate-share request that specifies the time
-  range of the reports in the batch.
+: A parameter of a query issued by the Collector that specifies the time range
+  of the reports in the batch.
 
 Client:
 : The endpoint from which a user sends data to be aggregated, e.g., a web
-  browser.
+  browser. As defined in {{!VDAF}}.
```

Comment:
This doesn't seem right. There's more to a DAP client than VDAF's definition of a client, so perhaps rephrase to explain that a DAP client is also a VDAF client.

Additionally, while you're here, referring to the client as an "endpoint" is confusing.

Reply:
Are you alright with "A party that uploads a report?" I believe the "endpoint" language stems from the distinction we made early on between "user" and "client".

Reply:
My problem with "endpoint" is that I read it to mean "a URL relative to which an API can be accessed", which isn't right in the case of the client. "A party that uploads a report" sounds fine.